pkgs/by-name/dm/dmarc-report-converter/package.nix

```diff
@@ -0,0 +1,29 @@
+{ buildGoModule, fetchFromGitHub, lib, pkgs }:
+
+buildGoModule rec {
+  pname = "dmarc-report-converter";
+  version = "0.6.5";
+
+  src = fetchFromGitHub {
+    owner = "tierpod";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-4rAQhZmqYldilCKomBfuyqS0vcUg5yS4nqp84XSjam4=";
+  };
+
```

Comment:
`--version` isn't showing a version, you probably have to add that with ldflags

Reply:
Can you please explain this a bit further or show me an example?

Reply:
![image](https://github.com/NixOS/nixpkgs/assets/80165193/9c26f381-3bbc-48cb-b99d-cd74b3a236c7)

the version string here is missing some go packages like for example: `pkgs/tools/networking/containerlab/default.nix` are adding this using ldflags.

Reply:
Done.